src/components/dashboard/Marketplace.js

```diff
@@ -59,7 +59,7 @@ const MarketTab = props => {
 
```

Comment:
this component was left from eToro, now we have own GoodMarket opened in a new tab/browser so it could be totally removed. I removed it during GoodMarket button implementation, probably it was wrongly kept during merge  RN to the master